### PR TITLE
added demo Networkpolicy

### PR DESCRIPTION
**Pull request by ch-stark**

### CODE REVIEW

**Comment by ch-stark** [comment]
see issue: https://github.com/open-cluster-management/policy-collection/issues/17

File: community/CM-Configuration-Management/policy-egress-firewall-sample.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: egress-example
+  namespace: default
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-egress-firewall-sample.yaml]: remove hardcoded ns

File: community/CM-Configuration-Management/policy-egress-firewall-sample.yaml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                        dnsName: analytics.example.com
+                    - type: Deny
+                      to: 
+                        cidrSelector: 0.0.0.0/0
+---
+apiVersion: policy.open-cluster-management.io/v1
+kind: PlacementBinding
+metadata:
+  name: binding-egress-example
+  namespace: default
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-egress-firewall-sample.yaml]: remove hardcoded ns

File: community/CM-Configuration-Management/policy-egress-firewall-sample.yaml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+subjects:
+- name: egress-example
+  kind: Policy
+  apiGroup: policy.open-cluster-management.io
+---
+apiVersion: apps.open-cluster-management.io/v1
+kind: PlacementRule
+metadata:
+  name: placement-egress-example
+  namespace: default
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-egress-firewall-sample.yaml]: remove hardcoded ns

**Comment by ycao56** [comment]
@ch-stark can you resolve merge conflicts and remove ns in all yamls?

**Comment by ch-stark** [comment]
@ycao56 will do, tnx

**Review by dockerymick — APPROVED** [review]

**Review by ycao56 — APPROVED** [review]

**Pull request merged**